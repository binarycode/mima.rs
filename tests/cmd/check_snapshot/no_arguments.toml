args = 'check-snapshot'
status.code = 2
