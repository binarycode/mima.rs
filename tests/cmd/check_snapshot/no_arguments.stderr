error: The following required arguments were not provided:
    <GUEST_ID>
    <SNAPSHOT_ID>

USAGE:
    mima check-snapshot <GUEST_ID> <SNAPSHOT_ID>

For more information try --help
